doc/bliss.xml

```diff
@@ -11,83 +12,58 @@
 <#GAPDoc Label="AutomorphismGroupDigraph">
 <ManSection>
   <Attr Name="AutomorphismGroup" Label="for a digraph" Arg="digraph"/>
-  <Oper Name="AutomorphismGroup" Label="for a digraph and a homogeneous list"
-    Arg="digraph, colors"/>
-  <Returns>A permutation group.</Returns>
+  <Returns>A permutation group, or a direct product of permutation
```

Comment:
Yes. `AutomorphismGroup` for a multidigraph previously had its own manual section, but I combined it with the section for a digraph without multiple edges (but without colours). The returns bit for the multidigraph section said "A direct product of permutation groups", so I just kept that bit.